.automatedtesting.json

```diff
@@ -12,10 +12,5 @@
     "ENABLE_GEMS": [
     ],
     "REPOS": [
-        {
-            "NAME": "o3de-extras",
-            "URL": "https://github.com/o3de/o3de-extras.git",
-            "BRANCH": "development"
-        }
```

Comment:
We don't want to merge this or the .gitignore change over to development.

The purpose of these changes is to prevent o3de stabilization from building against Gems from the o3de-extras `development` branch

Calling @amzn-changml

Reply:
The other changes in this PR are good though.

Reply:
Great catch, thanks! I'll remove that commit

Reply:
I've reverted those 2 commits to restore the o3de-extras repo for testing.